### PR TITLE
Update image generation functions

### PR DESCRIPTION
**Pull request by rich-iannone**
This PR provides some much-needed bug fixes and enhancements for the set of image generation functions (e.g, `local_image()`, `web_image()`, etc.).

### CODE REVIEW

File: R/image.R [hunk 1]
```diff
@@ -284,24 +273,41 @@ ggplot_image <- function(plot_object,
...
+        plot = plot_object[[x]],
+        device = "png",
+        dpi = 100,
+        width = 5 * aspect_ratio,
+        height = 5
+      )
 
-  file.remove("temp_ggplot.png")
+      # Wait longer for file to be written on async filesystems
+      Sys.sleep(0.5)
```
Comment (jcheng5) [inline R/image.R:301]:
```suggestion
      # Wait longer for file to be written on async filesystems
      Sys.sleep(0.5)
```

File: R/image.R [hunk 2]
```diff
@@ -284,24 +273,41 @@ ggplot_image <- function(plot_object,
...
 
-  image_html
+      image_html <-
+        local_image(filename = filename, height = height)
+
+      file.remove(filename)
```
Comment (jcheng5) [inline R/image.R:306]: This should be done in an on.exit, right after the ggsave

File: tests/testthat/test-image.R [hunk 3]
```diff
@@ -1,76 +1,125 @@
...
+    web_image(
+      url = c("example.com/img1.png", "example.com/img2.png"),
+      height = px(200)
+    ),
+    paste0("<img src=\"example.com/img", 1:2, ".png\" style=\"height:200px;\">")
+  )
+})
+
+test_that("the `ggplot_image()` function works correctly", {
+
```
Comment (jcheng5) [inline tests/testthat/test-image.R:99]: Can we just skip on CI here, not at the top of this file?

File: R/image.R (hunk 1 above)
Comment (rich-iannone) [reply R/image.R:301]: Removed this line.

File: R/image.R (hunk 2 above)
Comment (rich-iannone) [reply R/image.R:306]: Now added that in.

File: tests/testthat/test-image.R (hunk 3 above)
Comment (rich-iannone) [reply tests/testthat/test-image.R:99]: Now done.

**Review by jcheng5 — DISMISSED** [review]

**Review by jcheng5 — DISMISSED** [review]

**Review by jcheng5 — APPROVED** [review]

**Pull request merged**